bridge/node/js/import-bundle-builder.ts

```diff
@@ -25,9 +25,9 @@ export class ImportBundleBuilder {
     return new ImportBundleBuilder({ ...this.values, schemaId });
   }
 
-  public withGraphKeyPair(keyType: GraphKeyType, publicKey: Uint8Array, secretKey: Uint8Array): ImportBundleBuilder {
```

Comment:
nit: would be nice to have 2 variants: `withGraphKeyPair(key: GraphKeyPair)` and `withGraphKeyPairs(keys: GraphKeyPair[])`

Reply:
yeah thats cool i like it, and wont break any consumer if any